PowervRA/Functions/Public/config-service/Set-vRAHostname.psm1

```diff
@@ -0,0 +1,83 @@
+configure hostname
+
+get the node id of the appliance
+
+name is mandatory
+
+function Set-vRAHostname {
+<#
+    .SYNOPSIS
+    Set the hostname value used to resolve the application.
+    
+    .DESCRIPTION
+    This does not change the appliance hostname, only the hostname alias (typically the load balancer VIP FQDN.)
+
+    .PARAMETER Name
+    The name of the node on which to execute the command.
+
+    .PARAMETER Hostname
+    The desired hostname value.
+    
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.Management.Automation.PSObject.
+
+    .EXAMPLE 
+    Set-vRAHostname -Name vra-01a.lab.local -Hostname vra.lab.local
+
+#>
+[CmdletBinding(DefaultParameterSetName="Standard")][OutputType('System.Management.Automation.PSObject')]
```

Comment:
Ah, sorry... this function shouldn't be part of this pull request. I'll get it working properly and update this and some other pieces.